src/MongoDB/Server.c

```diff
@@ -295,7 +295,7 @@ static PHP_METHOD(Server, getTags)
 } /* }}} */
 
 /* {{{ proto array MongoDB\Driver\Server::getInfo()
-   Returns the last isMaster result document for this Server */
+   Returns the last hello response for this Server */
```

Comment:
Reminder: we'll need a separate PR to update the example in https://www.php.net/manual/en/mongodb-driver-server.getinfo.php. Users will certainly continue to get `ismaster` from older server versions, but I assume we can update the example to include the response from a 5.0+ server.

Reply:
Just saw https://github.com/mongodb/mongo-php-library/pull/828/files#diff-17b7c449b8afb0e8b5e94d80a7a42dd52689f9347aa1d18bd40f6ec30020b56dR65.

Perhaps the doc change is just a matter of swapping `ismaster` for `isWritablePrimary`.

Reply:
PHP docs are being handled in https://github.com/alcaeus/doc-en/pull/2